apple_stocks_python/apple_stock.py

```diff
@@ -0,0 +1,35 @@
+
+def get_max_profit_yesterday(prices):
+    if len(prices) <= 1:
+        return 0
+
+    minimum_price = prices[0]
+    max_profit = 0 # profit can't be negative
+
+    for current_price in prices:
+        minimum_price = min(current_price, minimum_price)
+        current_profit = current_price - minimum_price
+        max_profit = max(max_profit, current_profit)
+
+    return max_profit
+
+
+def test_run(input_arr):
```

Comment:
Instead of the user manually looking at the output to verify if the test is correct, expect what the output should be, so that we can automate it.